src/component/movies/MovieLikes.js

```diff
@@ -0,0 +1,56 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { addLike, removeLike, flipLike } from '../../store/actions/MovieActions';
+import '../../styles/css/movies/movie_like.css'
+
+class MovieLikes extends Component {
+  
+    performLike = (like) => {
+        if (this.props.movie.user_liked_or_disliked === 0) {
+            this.props.addLike({movie: this.props.movie.id, like: like});
+        }
+        else if (this.props.movie.user_liked_or_disliked === like) {
+            this.props.removeLike({movie: this.props.movie.id, like: like});
```

Comment:
sobizrom da nam je kao sto si i sam rekao movie - user unique na nivou tabele, onda nema potrebe da saljemo like.

Reply:
Ja moram poslati like parametar posto saljem ili 1 ako je like ili -1 ako je dislike

Reply:
ali ne moze svakako korisnik imati i jedno i drugo, tako da uvijek ces imati u bazi ili jedan slucaj ili drugi, sto znaci remove ce obrisati nezavisno od slucaja. Ne vidim da tu imas neki problem ako tako odradis

Reply:
Nisam ja bio razumeo sta si tacno bio mislio. Sad mi je jasno tek da si konkretno na to mislio.
Ja to ne saljem na BE, vec samo do movie reducer-a provlacim to, da bih tamo znao da li da smanjim broj like-ova ili dislike-ova

Reply:
pa i to mozes onda da odradis tako sto imas informaciju o tome sta je korisnik uradio sa tog posta, sto vec imas na reducer-u, zar ne ?

Reply:
Da zapravo imam informaciju sta je prethodno korisnik dao, pa to smanjim. Nije mi to na pamet, ispravicu da tako radi onda.